src/main/resources/run-docker-container.sh

```diff
@@ -3,6 +3,7 @@
 exec java \
     -DLOG_LEVEL=${LOG_LEVEL} \
     -Ds3proxy.endpoint=http://0.0.0.0:80 \
+    -Ds3proxy.virtual-host=${S3PROXY_VIRTUALHOST} \
```

Comment:
@jim-slattery-rs 6b30e6612779e54c1741e522a09d6bd6b2dd31b4 should fix this and Travis updated the Docker image.  Sorry about the breakage and thanks for your feedback!